libs/dal/src/repositories/integration/integration.entity.ts

```diff
@@ -48,6 +50,14 @@ export class IntegrationEntity {
 
   active: boolean;
 
+  name?: string;
+
+  identifier?: string;
+
+  kind?: ProviderKindEnum;
```

Comment:
Typing as `ProviderKindEnum` in case in the future we have `kinds` in other entities to help the readability and searcheability.

Reply:
I am not sure about 'kind'. What do you think about 'category' ? Both don't feel completely right🤷🏼‍♀️

Reply:
A small question regarding the `kind` property, if we are doing the "conditions" array, doesn't it mean that the kinds can be combined in a more dynamic way inside the the `conditions` array. Meaning that you can have the following dynamic conditions:
- TenantId = 123
- AND
- Environment = production
- AND
- Workflow = 123

So in this case it's either the "kind" or "conditions" approach, not the two of them combined. What are your thoughts on this, maybe I'm missing something

Reply:
@ainouzgali `kind` and `category` are synonims in a way. In the technical design I chose `kind` as it is used to define entities in places like GCP and I come from working a lot with GCP.

Reply:
> A small question regarding the `kind` property, if we are doing the "conditions" array, doesn't it mean that the kinds can be combined in a more dynamic way inside the the `conditions` array. Meaning that you can have the following dynamic conditions:
> 
> * TenantId = 123
> * AND
> * Environment = production
> * AND
> * Workflow = 123
> 
> So in this case it's either the "kind" or "conditions" approach, not the two of them combined. What are your thoughts on this, maybe I'm missing something

Ok, I wasn't aware that we totally ditched the `kind` property. Makes no sense if we are going to go `conditions` approach.